.teamcity/_self/projects/WebApp.kt

```diff
@@ -125,8 +125,19 @@ object BuildDockerImage : BuildType({
 				export GH_TOKEN="%matticbot_oauth_token%"
 				chmod +x ./bin/add-pr-comment.sh
 				./bin/add-pr-comment.sh "%teamcity.build.branch%" "calypso-live" <<- EOF || true
-				Link to Calypso live: https://calypso.live?image=registry.a8c.com/calypso/app:build-%build.number%
-				Link to Jetpack Cloud live: https://calypso.live?image=registry.a8c.com/calypso/app:build-%build.number%&env=jetpack
+				<table>
+					<tr><td>
+					Link to <a href="https://calypso.live?image=registry.a8c.com/calypso/app:build-%build.number%">Calypso live (%build.number%)</a>
```

Comment:
This will make harder to copy/paste the link.

Reply:
I think it's a good trade-off in order to tidy up the comment. You can still right-click and "Copy Link" if needed. I guess most people will just click it anyway (or copy it from the QR code). Did you often copy it? Trying to better understand the use-case!

Reply:
We could maybe update the comment to end up looking like 
<img width="400" alt="Screen Shot 2021-12-29 at 11 08 34 AM" src="https://user-images.githubusercontent.com/115071/147695550-a7584751-6ba5-4a65-a2df-9bd05b66ca0c.png">
but if you ask me that takes up too much vertical space.